src/main/java/com/conveyal/datatools/manager/controllers/api/OrganizationController.java

```diff
@@ -69,43 +71,14 @@ public static Organization updateOrganization (Request req, Response res) throws
         requestOrganizationById(req);
         Organization organization = Persistence.organizations.update(organizationId, req.body());
 
-        // FIXME: Add back in hook after organization is updated.
```

Comment:
I don't think it is good practice to leave large sections of commented-out code in a repository.  If you need this, you can always go back to a previous commit and copy it back in.

Reply:
Yes, I can agree with that, but I'd rather not completely wipe out all traces of this in the current codebase.  I will create an issue for this, but I think if there is a `FIXME` in the code you shouldn't just wipe it out without any questions asked.

Reply:
Yes, though it's understandable as a best practice to avoid leaving blocks of commented code in a repo, it's not sufficient that we can just go back and find it. Remembering that something has already been implemented, knowing where it was implemented, and then going back to find the commit where it was erased might be nearly impossible. We should only delete such blocks of code if there is no information loss, including information about the existence of a problem or a proposed solution to that problem. Creating a ticket as @landonreed did is probably the best option.